moduleframework/common.py

```diff
@@ -1,4 +1,4 @@
-#
+# -*- coding: utf-8 -*-
 # Meta test family (MTF) is a tool to test components of a modular Fedora:
```

Comment:
I would prefer to remove it. It tests also containers. Not or modular Fedora. The line below too.